_posts/2017-08-13-this-week-in-rustfest-6-cfp-finalisation.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "This week in RustFest 6: CFP finalisation"
```

Comment:
I'm not sure "CFP finalization" is very accurate. How about "Talk selection" or "Schedule finalization?". Finalizing the _call_ for proposals is not the same as selecting talks, no?